official-package/aws-organizations/crowdstrike-ssm-distributor-deploy.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+print("Installing crowdstrike-falconpy and required dependencies..")
+os.system("pip3 install -r requirements.txt -t ./source/python")
```

Comment:
Looks like the requirements.txt file is empty.

Also are we only using python for running SAM commands? If so maybe we just give them the SAM commands to run since there are only 2. This prevents the need for them to have a python interpreter on their system.